feature-detects/workers/workertypeoption.js

```diff
@@ -0,0 +1,44 @@
+/*!
+{
+  "name": "worker type option test",
+  "property": "workertypeoption",
+  "caniuse":"mdn-api_worker_worker_ecmascript_modules",
+  "tags": ["web worker type options", "web worker"],
+  "builderAliases": ["worker_type_options"],
+  "authors": ["Debadutta Panda"],
+  "notes": [{
+    "name": "MDN Docs",
+    "href": "https://developer.mozilla.org/en-US/docs/Web/API/Worker/Worker"
+  }]
+}
+!*/
+/* DOC
+Detect working status of all Workeroptions
+*/
+/**
```

Comment:
again, keep the doc comments in one block

Reply:
Thank you for addressing the issue is resolved 👍